cmd/constants/marketmaps/markets.go

```diff
@@ -6354,67 +6259,6 @@ var (
 			  }
 			]
 		  },
-		  "RNDR/USD": {
```

Comment:
removed the dead feeds.  Validation fails when they are here and passing when they are gone